index.d.ts

```diff
@@ -109,9 +117,11 @@ export interface Unsubscribe {
  * There should only be a single store in a Redux app, as the composition
  * happens on the reducer level.
  *
- * @template S State object type.
+ * @template S The type of state held by this store.
+ * @template A the type of actions which may be dispatched by this store.
+ * @template N The type of non-actions which may be dispatched by this store.
  */
-export interface Store<S> {
+export interface Store<S = {}, A extends Action = Action, N = never> {
```

Comment:
Again, the default for `S` is unclear.

Reply:
See previous comment... if that doesn't sway you, I'll remove the default.